src/main/java/org/gridsuite/modification/dto/CurrentLimitsInfos.java

```diff
@@ -27,6 +25,9 @@
 @Schema(description = "Current Limits")
 public class CurrentLimitsInfos {
 
+    @Schema(description = "Operational limit group name")
+    private String operationalLimitGroupId;
```

Comment:
operationalLimitsGroupId ?

Reply:
also change the description

Reply:
This class member has been removed. But yep the 's' was missing.